language/documentation/examples/diem-framework/move-packages/DPN/sources/AccountAdministrationScripts.move

```diff
@@ -356,7 +356,7 @@ module DiemFramework::AccountAdministrationScripts {
 
         /// **Access Control:**
         /// Only the Diem Root account can process the admin scripts [[H9]][PERMISSION].
-        requires Roles::has_diem_root_role(dr_account); /// This is ensured by DiemAccount::writeset_prologue.
+        requires Roles::has_diem_root_role(dr_account); // This is ensured by DiemAccount::writeset_prologue.
```

Comment:
This was a `DocComment` which doesn't match any `DocComment` described language item. It didn't throw warning of `InvalidDocComment`. I made some of the same doc comments changes below.